lintro/tools/core/tool_installer.py

```
@@ -0,0 +1,426 @@
+        logger.info(f"Installing {tool.name}: {command}")
+        start = time.monotonic()
+
+        try:
+            # For binary tools, try to use install-tools.sh if available
+            if tool.install_type == "binary":
+                result = self._install_via_script(tool)
+                if result:
+                    return result
+
+            # Otherwise run the command directly
+            proc = subprocess.run(
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)